tests/webhook_test.go

```diff
@@ -182,11 +182,123 @@ func (c *ClientTests) TestValidateWebhookMissingSecret() {
 	client := c.TestClient()
 	assert := c.Assert()
 
-	webhookSecret := "123"
+	webhookSecret := c.fixture.WebhookSecret()
 	headers := map[string]interface{}{
 		"some-header": "some-value",
 	}
 
 	_, err := client.ValidateWebhook(c.fixture.EventBody(), headers, webhookSecret)
 	assert.Error(err)
 }
+
+func (c *ClientTests) TestWebhookCreateWithCustomHeaders() {
+	client := c.TestClient()
+	assert, require := c.Assert(), c.Require()
+
+	customHeaders, err := c.fixture.WebhookCustomHeaders()
+	require.NoError(err)
+
+	webhook, err := client.CreateWebhookWithDetails(
+		&easypost.CreateUpdateWebhookOptions{
+			URL:           c.fixture.WebhookUrl(),
+			CustomHeaders: customHeaders,
```

Comment:
suggestion: I'd prefer that the `CustomHeaders` get passed in the `CreateWebhookWithDetails` test instead of needing a separate test here (see other client libraries). We are ultimately looking to keep the test suite as simple as possible and our create tests usually attempt to pass in happy-path params as a single test. Same for the update test.

So ultimately we'd want no new tests added, the secret and custom header params to be added to the existing create and update tests, and asserting that the custom header name/value is in the response (see other PRs)